src/components/EachBook.js

```diff
@@ -3,29 +3,63 @@ import { useDispatch } from 'react-redux';
 import { removeBook } from '../redux/books/booksSlice';
 
 function EachBook({
-  title, author, category, id,
+  book, id,
 }) {
   const dispatch = useDispatch();
+
   return (
     <>
-      <div className="inner-container">
-        <p>{category}</p>
-        <h2>{title}</h2>
-        <p>{author}</p>
-        <div className="buttons">
-          <button type="button">Comments</button>
-          <button type="button" onClick={() => dispatch(removeBook(id))}>Remove</button>
-          <button type="button">Edit</button>
+      <section className="inner-container">
+        <div id="box each-book">
+          <p className="grey">category</p>
+          <h2>{book.title}</h2>
+          <p className="author">{book.author}</p>
+          <div className="margin-buttons buttons">
+            <button type="button">Comments</button>
+            <button type="button" onClick={() => { dispatch(removeBook(id)); }}>Remove</button>
+            <button type="button">Edit</button>
+          </div>
+        </div>
```

Comment:
- [ ] Your final design should match zeplin's design (this includes all elements + the same styling). Kindly follow the given design.

Given design
![image](https://github.com/joyapisi/bookstore-app/assets/81932478/fd7fc62a-825a-4bf8-a4c0-bab173e594d6)

Your design
![image](https://github.com/joyapisi/bookstore-app/assets/81932478/69cb98ac-cf44-453e-a353-97c1e0241c21)

- _Not the difference between your navigation and the given design's navigation, `Home` and `Categories` links should be aligned to the left, and to the right there should be an Icon_
- _Kindly give the `UPDATE PROGRES` button some padding as to match the given design.
- _The `ADD NEW BOOK` section should also follow the given layout, by spacing the input fields and the `ADD BOOK` button as is in the given design._
![image](https://github.com/joyapisi/bookstore-app/assets/81932478/68fdc42b-37da-4e18-8459-19c6f64b5045)